docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Messaging.adoc

```diff
@@ -558,6 +558,41 @@ CLI:
 }
 ----
 
+[[jms-bridge-statistics]]
+=== Statistics of a JMS Bridge
+
+Currently two statistics are available on a JMS bridge: the number of processed messages and the number of aborted/rolled back messages.
+Those are available with the following command :
+
+[source, ruby]
+----
+/subsystem=messaging/jms-bridge=myBridge:read-attribute(name=message-count)
+{
+    "outcome" => "success",
+    "result" => [{
+        "address" => [
+            ("subsystem" => "messaging"),
+            ("jms-bridge" => "myBridge")
+        ],
+        "outcome" => "success",
+        "result" => 0L
+    }]
+}
```

Comment:
This should not be the structure of the response. I believe it is simply:

```
{
  "outcome" => "success",
  "result" => 0L
}
```

What you show above is what you'd see if the request had been for jms-bridge=*, not for a specific bridge

/subsystem=messaging/jms-bridge=*:read-attribute(name=message-count)